website/docs/module_ftrack.md

```diff
@@ -72,7 +72,7 @@ We do not recommend setting your Ftrack user and api key environments in a persi
 
 ### Where to run event server
 
-We recommend you to run event server on stable server machine with ability to connect to Avalon database and Ftrack web server. Best practice we recommend is to run event server as service. It can be Windows or Linux. 
+We recommend you to run event server on stable server machine with ability to connect to Avalon database and Ftrack web server. Best practice we recommend is to run event server as service. It can be Windows or Linux.
```

Comment:
Does this make more sense to rename to "OpenPype" database? Or since the database is named `avalon` in mongodb we'd like to keep as is?

```suggestion
We recommend you to run event server on stable server machine with ability to connect to OpenPype database and Ftrack web server. Best practice we recommend is to run event server as service. It can be Windows or Linux.
```